cupyx/scipy/ndimage/filters.py

```diff
@@ -151,6 +151,8 @@ def _generate_correlete_kernel(ndim, mode, cval, xshape, wshape, origin):
     out_params = 'Y y'
 
     ops = []
+    ops.append('X* _x = (X*)&(x[0]);')
```

Comment:
Can you change the variable prefix?
We are working on some coding guidelines for CuPy and one of them will be not allowing '_' to prefix local variables.
Thanks!

Reply:
Got it!
BTW, is it OK to use '_' as suffix (^^).